packages/lit/src/index.all.ts

```diff
@@ -36,6 +36,7 @@ export {
   svg as staticSvg,
   unsafeStatic,
   withStatic,
+  StaticValue,
```

Comment:
If I understand correctly this file is for the prebuilt [JS bundle](https://lit.dev/docs/getting-started/#use-bundles) of Lit. Not sure if exporting an interface here is required which I assume gets removed from the JS bundle, although not a blocker.